docs/src/literate/incompressible_elasticity.jl

```diff
@@ -157,7 +157,7 @@ function assemble_up!(Ke, fe, cell, cellvalues_u, cellvalues_p, facevalues_u, gr
     ## We loop over all the faces in the cell, then check if the face
     ## is in our `"traction"` faceset.
     @inbounds for face in 1:nfaces(cell)
-        if onboundary(cell, face) && (cellid(cell), face) ∈ getfaceset(grid, "traction")
+        if onboundary(cell, face) && FaceIndex(cellid(cell), face) ∈ getfaceset(grid, "traction")
```

Comment:
Hm, I guess this is pretty breaking. Perhaps we can leave `FaceIndex ` as `const FaceIndex = Tuple{Int, Int}` but use struct for the others?